ui/app/builds/directives/pnc-log-streamer/pncLogStreamer.js

```diff
@@ -34,7 +34,7 @@
       // --------------------
 
       $ctrl.$onInit = () => {
-        const bifrostUrl = new URL(pncProperties.externalBifrostUrl);
+        const bifrostUrl = new URL(pncProperties.bifrostWsUrl);
         $ctrl.bifrostHost = bifrostUrl.host;
```

Comment:
Why `getBifrostHost` below (in `bifrost/bifrost.module.js`) is not used? Or should the method below be deleted?

And again, `$ctrl.bifrostWsHost` should be preferred (make sure to update all required places)

Reply:
`Why getBifrostHost below (in bifrost/bifrost.module.js) is not used? ` This was how it was implemented before I touch the code, I will change it as you wish.

Reply:
Is there any difference between those use-cases? You know bifrost the best, I will leave it up to you to choose what's the best.

Reply:
not at all, I think it was just Alex used two different ways to implement the same thing. I was trying not to touch the old code but improvements can also be feasible. I have fixed in the latest commit.